### DISCUSSION TITLE
My account was suspended without any notice or support from github

### DISCUSSION THREAD

**Discussion by Z-code-helper** [community/community#136624] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My github account was suspended suddenly and without any notice, which has seriously affected my work.
I submitted a support ticket through Github's support page but it's been more than 7 days and I still haven't received any response or support.

My entire work has been suspended and my repositories are no longer accessible. There is no way to restore or at least what caused this terrible situation from github, which makes me worried about my future creations if this situation continues.

I have read and found that not only me but many people out there are also facing the same situation as me, I don't know how github allows this to happen so often, are we using the free plan and not worthy of their support as businesses?

**Comment by Dhruvnet** [comment, marked as answer]
It's really frustrating to experience an account suspension, especially when it impacts your work so heavily. Here are a few steps you can take to address the situation:

Follow Up on the Support Ticket
Since it's been over 7 days without a response, try following up on your support ticket. You can reply to the original ticket to keep it active in the support queue.
Be sure to clearly state the urgency of the situation, the impact it's having on your work, and request an escalation if possible.
Contact GitHub Support via Other Channels
Twitter: Sometimes, social media can expedite the process. Consider tweeting at [GitHub Support](https://twitter.com/GitHubSupport) with a polite request for assistance and mention your ticket number.
Community Forums: You might try posting in the [GitHub Community Forums](https://github.community/) to see if you can get any advice or assistance from other users or GitHub staff.
Check GitHub’s Terms of Service
Review GitHub’s [Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service) to ensure you haven't unintentionally violated any policies. If you believe the suspension was a mistake, clearly explain why in your communications.
Backup Plan for Future Work
While waiting for a resolution, consider setting up a backup system for your work. Using platforms like [GitLab](https://gitlab.com/) or [Bitbucket](https://bitbucket.org/) can provide an alternative.
Regularly back up your repositories locally or on another service to avoid disruptions in the future.
Seek Legal Advice
If the suspended account is critical to your business or career and GitHub remains unresponsive, seeking legal advice might be a necessary step. A legal professional can help you understand your rights and potentially assist in getting a response from GitHub.
Learn from the Community
Research other cases where users have experienced similar issues. Sometimes, community-driven solutions or shared experiences can provide insights or alternative contact methods.
Consider Paid Support
If you’re on a free plan and your work is crucial, consider upgrading to a paid plan. Paid plans often come with more robust support options. You might also explore if there are expedited support options available for urgent issues.
If you continue to face issues and don't receive a response, publicizing the problem on platforms like LinkedIn, Medium, or industry forums might also draw attention to your case, potentially prompting faster action from GitHub.
